SharedPackages/BrowserServicesKit/Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -1366,22 +1372,26 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
     @available(iOS 17, *)
     @MainActor
-    public func handleShutDown() async throws {
+    public func handleShutDown(dueTo error: Error) async throws {
+        enum ShutDownError: Error {
+            case noManagerFound
+        }
```

Comment:
Not used anywhere else so I declared this within this method.  This is only useful for pixel information at this point and is not intended to be managed.